src/cy_serial_bridge/driver.py

```diff
@@ -364,6 +364,80 @@ def read_user_flash(self, addr: int, size: int) -> bytearray:
             result_bytes.extend(page_bytes)
 
         return result_bytes
+    
+
+    def is_gpio_output(self, gpio_nr: int) -> bool:
```

Comment:
Could we remove these functions for now as they are not implemented?  We will of course add them when they are implemented.